types/koa-jwt/index.d.ts

```diff
@@ -7,9 +7,11 @@ import Koa = require("koa");
 
 export = jwt;
 
-declare function jwt(options: jwt.Options): Koa.Middleware;
+declare function jwt(options: jwt.Options): jwt.Middleware;
 
 declare namespace jwt {
+    type booleanReturn = (args?: any) => boolean;
```

Comment:
This doesn't deserve its own alias -- just use `custom?(arg: any): boolean;`
(better, provide a type for `arg`)